recipe/eqt_env.yml

```diff
@@ -1,9 +1,8 @@
 name: eqt_env
 channels:
   - conda-forge
-  - https://software.repos.intel.com/python/conda
 dependencies:
   - python
```

Comment:
if eqt only works with python 3.12 let's add the constrain in the requirementsfor now, we cannot risk to break iDVC and CIL-GUI. This is a temporary solution until we upgrade to pyside6. Perhaps we can add a comment in the recipe stating this.

@paskino do you agree?

Reply:
I will limit the conda recipe to install a Python version <3.12 (I have tested and the examples work up until 3.11.10, any version of 3.12 does not behave correctly).

Should I add a comment to the recipe explaining the decision, or is it enough to have it in the change log?

Reply:
Thank you! Modifying the change log is enough. If you link the PR (this) we will be able to retrieve in the future why we did a specific change. The goal is to remove this constrain in the future, probably by upgrading to pyside6

Reply:
Please add any test you did and any info you have on the topic in the PR textbox